app/assets/stylesheets/buttons.scss

```diff
@@ -148,18 +148,18 @@
     cursor: default !important;
 
     &.positive {
-      background: $light-green;
-      color: $green;
+      background: $gray-tea-green;
+      color: $sea-green;
     }
 
     &.neutral {
-      background: $light-blue;
-      color: $blue;
+      background: $pale-cornflower;
+      color: $steel-blue;
     }
 
     &.negative {
-      background: $light-red;
-      color: $red;
+      background: $apricot;
+      color: $cinnamon;
     }
   }
 
```

Comment:
I have proposition using the next code instead of that. Your code is nice, but we have a special buttons which describing in the Design System Vibe Report Documents and which was created for special apply
```
.btn {
  &.btn-bubbles {
    .edit-icon {
      position: relative;
      left: 229px;
      top: -17px;
      cursor: pointer;
    }
    &:active{
      margin:0 !important;
    }
  &.wb2 {
    border: 2.25px solid $black;
  }
  }
}
```
and change class in the const  `BigBtnEmotion` on this
`className={`${addClass} btn btn-bubbles wb2 ${emotion.category}`  `
I understand in this button have animation but we can to switch off now or later if that will not be  like our designer Mike

Reply:
@VitaliiShevchenko I'm not sure if this makes sense in this way, I'd prefer to leave the current implementation